styles/_theme.scss

```diff
@@ -26,6 +26,11 @@ button,
   input[type="radio"]:checked + label:before {
     box-shadow: inset 0 0 0 3px $secondary;
   }
+
+  &#pageTabBar:before {
+    // the chat button cutout shadow fills the rest of the tab bar with color
+    box-shadow: 0 0 0 200vw $primary;
```

Comment:
The peak through window for the close button fills the page tab with it's drop shadow. 

I made a demo of the effect here: https://jsfiddle.net/jjeffryes/8ew04tz5/